wled00/data/settings_sync.htm

```diff
@@ -127,7 +127,7 @@ <h3>WLED Broadcast</h3>
 Send Alexa notifications: <input type="checkbox" name="SA"><br>
 Send Philips Hue change notifications: <input type="checkbox" name="SH"><br>
 Send Macro notifications: <input type="checkbox" name="SM"><br>
-Number of UDP Messages: <input name="UR" type="number" min="1" max="255" class="d5" required><br><br>
+UDP packet retransmissions: <input name="UR" type="number" min="1" max="30" class="d5" required><br><br>
```

Comment:
why minimum of 1? perhaps I do not want retransmissions. 😄

Reply:
Had my blinders on.....